dashboard/api.py

```diff
@@ -729,7 +729,7 @@ def refresh_user_data(user_id, provider=BACKEND_EDX_ORG):
         log.exception("Unable to create an edX client object for student %s", user.username)
         return
 
-    for cache_type in CachedEdxDataApi.SUPPORTED_CACHES:
+    for cache_type in CachedEdxDataApi.EDX_SUPPORTED_CACHES:
```

Comment:
Should this be this instead?
```suggestion
    for cache_type in CachedEdxDataApi.CACHE_TYPES_BACKEND[provider]:
```

Reply:
this function is only used by the batch_update_user_data task, so I thought to leave it as is. But I think I can update it.